pennylane/plugins/default_qubit.py

```diff
@@ -131,7 +140,7 @@ def apply(self, operations, rotations=None, **kwargs):
 
     @property
     def state(self):
-        return self._pre_rotated_state
+        return self._pre_rotated_state.ravel()
```

Comment:
@johannesjmeyer What is the plan with using `ravel()`? Is it to treat the state as a tensor internally?

Reply:
Exactly.